src/ui/components/DialogSharedEntryBindings/DialogSharedEntryBindings.scss

```diff
@@ -0,0 +1,36 @@
+.dialog-shared-entries-binding {
+    width: 900px;
+    max-height: 90vh;
+    min-height: 590px;
+    height: 0;
+
+    &__body {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+    }
+
+    &__current-row {
+        margin-bottom: 16px;
+        &_divider {
```

Comment:
btw, we have Divider in uikit https://preview.gravity-ui.com/uikit/?path=/docs/components-utils-divider--docs